extensions/wikia/Recirculation/services/CakeRelatedContentService.php

```diff
@@ -29,9 +30,46 @@ public function getContentRelatedTo($title, $limit=5, $ignore=null) {
 		}
 
 		$api = $this->relatedContentApi();
-		
+
 		try {
 			$filteredRelatedContent = $api->getRelatedContentFromEntityName($title, $limit + 1, "true");
+            if (is_null($filteredRelatedContent)) {
+                WikiaLogger::instance()->warning("getRelatedContentFromEntityName failed to retrieve recommendations", [
```

Comment:
Also, the spacing seems off here. I'm not sure if the original file is correct and the additions are wrong, or vice versa.